lesson-outline.md

```diff
@@ -1,87 +1,56 @@
 	* What is a cloud?
-	* What is a cluster? Different cluster types
-	* Overview of services available (Compute Canada, Amazon EC2, etc.)
-
-## 2. Submitting / running jobs
-
-This lesson is hosted in the [hpc-intro](https://github.com/hpc-carpentry/hpc-intro) repo.
-
-* Scheduler - lesson will cover SLURM by default (which can also run PBS scripts/commands natively)
-
-	* Submitting jobs
-	* Checking status of jobs
-	* Deleting jobs
-	* Job size consequences
-	* GUI vs. batch programs (X-forwarding, SSH tunnels?)
-
-* Using software and environment modules
-* Playing friendly in the cluster
+    * user profiles (academic and/or commercial)
+    * story introduction
+
+# jobs
+
+* Scheduler - lesson will cover SLURM by default, other schedulers are planned.   
+  From [hpc-in-a-day](https://github.com/psteinb/hpc-in-a-day) we know that material comprising Slurm, PBS and LSF is possible, but hard already as e.g. PBS doesn't support direct dispatchment as `srun`.
+
+    * [12-cluster.md](https://github.com/hpc-carpentry/hpc-intro/tree/gh-pages/_episodes/12-cluster.md)
+        * logging in with ssh
+        * looking around (compare with your laptop)
+        * now what? Scheduling basics (why scheduling, what is a job), see for example [here](https://psteinb.github.io/hpc-in-a-day/02-01-batch-systems-101/) on trying to illustrate the fair share problem
+    
+    * [13-scheduler.md](https://github.com/hpc-carpentry/hpc-intro/tree/gh-pages/_episodes/13-scheduler.md)
+	    * submit a `echo Hello World` on the command line (`srun`, `bsub`, etc.)
+        * check the automagically log file for output
+        * submit a batch script (`sbatch`, `bsub`, etc)
+        * NB: do not dive into options yet
+        * for a starting point, see [the bottom half of this](https://psteinb.github.io/hpc-in-a-day/02-01-batch-systems-101/) for reference
+        
+    * `14-advanced-jobs.md` (doesn't exist yet)
+        * Checking status of jobs (`squeue`, `bjobs` etc.), explain different job states and relate to scheduler basics
+        * Cancelling/deleting a job (`scancel`, `bkill` etc.)
+        * Passing options to the scheduler (log files)
+        * Callout: Changing a job's name
+        * Optional Callout: Send an email once the job completes (not all sites support sending emails)
+        * for a starting point, see [this](https://psteinb.github.io/hpc-in-a-day/02-02-advanced-job-scheduling/) for reference
+        
+        
+    * `15-filesystem-zoo.md` (doesn't exist yet)
+        * execute a job that collects node information and stores the output to `/tmp`
+        * ask participants where the output went and why they can't see it
+        * execute a job that collects node information and stores the output to `/shared` or however your shared file system is called
+        * for a starting point, see [this](https://psteinb.github.io/hpc-in-a-day/02-03-shared-filesystem/) for reference
+        
```

Comment:
The filesystem zoo is different on every system, but there are some common themes. Would it be more fruitful to discuss Lustre *vs.* `/tmp`, and how to extract data from each, rather than belaboring the `-o` and `-D` SBATCH flags?

Reply:
all yes! ;) One could use `<sbatch|bsub|...> -o` for that matter to illustrate the effect of using `/tmp` versus anything else. In [hpc-in-a-day](https://psteinb.github.io/hpc-in-a-day/02-03-shared-filesystem/), I used a shell variable inside a job script. I had the feeling this made the learners more aware of it as something that an application can run into, rather than a feature of `<sbatch|bsub|...>` only.